soccer/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ set(ROBOCUP_LIB_SRC
     "motion/TrapezoidalMotion.cpp"
     "NewRefereeModule.cpp"
     "planning/CompositePath.cpp"
-    "planning/DirectTargetPathPlanner.cpp"
+    #"planning/DirectTargetPathPlanner.cpp"
```

Comment:
Yeah it unfortunately requires some merging since we're messing with the same code.  If we work on the same branch, we have the same conflicts, we just end up doing them in smaller pieces.  There aren't really less conflicts though.  If you want, we can wait to merge the TargetVelPlanner and EscapeObstaclesPlanner until after this PR and I'll just handle the merge conflicts that show up with those planners.